lib/Orchestrator.js

```diff
@@ -95,14 +96,14 @@ Orchestrator.prototype.addModule = function(taskName, dep) {
 
 
 	if (this.ctxResolver) {
-		ctx = this.ctxResolver(taskName);
+		this.ctx  = this.ctxResolver(taskName);
```

Comment:
module context should not override global context. When a context is resolved through ctxResolver passing a module name, the result context should be applied only to the module.

That is the reason, there is a global context (this.ctx) and a local context (ctx) at a module level are available. Currently none of our use case needs module context.